docs/source/share_kedro_viz.md

```diff
@@ -21,116 +21,64 @@ kedro new --starter=spaceflights-pandas
-
-![](./images/kedro_viz_share_credentials1.png)
-
-Create a user group from the IAM dashboard, ensuring the user group has full access to the AWS S3 policy.
-
-![](./images/kedro_viz_share_credentials2.png)
-
-For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
-
-Add the IAM user to the user group (this is only possible if the group has been created).
+There are two ways to publish and share Kedro-Viz:
 
-![](./images/kedro_viz_share_credentials3.png)
+1. **Platform-agnostic sharing with Kedro-Viz:** Using the kedro-viz build command, users can publish Kedro-Viz to any static website hosting platform such as GitHub pages, and share the URL generated. This is described in [Platform-agnostic sharing with Kedro-Viz](./platform_agnostic_sharing_with_kedro_viz)
 
-Select the user, then select `Create access key`. Follow the steps and create your keys.
+2. **Publish and share Kedro-Viz automatically:** You can automate the process of publishing and sharing your Kedro-Viz through AWS. Once the storage account and credentials are configured, users can then generate a shareable link from the Kedro-Viz UI using the 'Publish and share' button.
 
-![](./images/kedro_viz_share_credentials4.png)
-
-
-Once that's completed, you'll need to set your AWS credentials as environment variables in your terminal window, as shown below:
-
-```bash
-export AWS_ACCESS_KEY_ID="your_access_key_id"
-export AWS_SECRET_ACCESS_KEY="your_secret_access_key"
+```{note}
+From Kedro-Viz version 7.2.0, in addition to AWS, you will be able to publish and share your Kedro-Viz on Azure and GCP.
 ```
 
-For more information, see the official AWS documentation about [how to work with credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
-
-## Publish and share the project
+* [Publish and share Kedro-Viz on AWS](./publish_and_share_kedro_viz_on_aws)
+* [Publish and share Kedro-Viz on Azure](./publish_and_share_kedro_viz_on_azure)
+* [Publish and share Kedro-Viz on GCP](./publish_and_share_kedro_viz_on_gcp)
 
-You're now ready to publish and share your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
+```{toctree}
+:caption: Learn about Publish and Share Kedro-Viz
+:maxdepth: 2
 
-```bash
-kedro viz run
+platform_agnostic_sharing_with_kedro_viz
+publish_and_share_kedro_viz_on_aws
+publish_and_share_kedro_viz_on_azure
+publish_and_share_kedro_viz_on_gcp
 ```
 
-Click the **Publish and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
-
-Once those two details are complete, click **Publish**. A hosted, shareable URL will be returned to you after the process completes.
-
-Here's an example of the flow:
-
-![](./images/kedro-publish-share.gif)
+## Publish and share via CLI
```

Comment:
Can this be also part of the cloud provider page.

Reply:
yes even I was in two minds about this. Since CLI command is common to all cloud providers, I mentioned it here. But even for running the CLI command, the setup needs to be done specific to cloud provider. May be we can move this to cloud provider specific page